modules/administration/pages/confidential-computing.adoc

```diff
@@ -0,0 +1,112 @@
+[[confidential_computing]]
+= Confidential Computing 
+
+{CoCo} is a technology which allows protection of data in use by using hardware-based Trusted Execution Environment (TEE), the type of environments that provide increased level of security for data integrity, data confidentiality, and code integrity.
+
+
+
+== {CoCo} with {productname}
+
+The trustworthiness of the TTE is checked with the attestation process.
+{productname} can be used as an attestation server for the systems registered to it.
+It generates a report page for the systems which run in this mode.
+These systems need to be attested and checked on regular base.
+The history of the past checks is also stored and available per request.
+
+{CoCo} Attestation depends on the used hardware and environment where the attested systems are running on.
+
+
+
+== Requirements
+
+{CoCo} can be set up in an environment with the following characteristics:
+
+* Attested system (virtual machine) is *SLES15 SP6* and bootstrapped to {productname}
+* Hardware must have *AMD EPYC Milan CPU* or *AMD EPYC Genoa CPU*
+
+//// 
+This was from the testing guidelines, perhaps it is not needed in the documentation?
+* [literal]``kernel-default`` must be installed instead of [literal]``kernel-default-base``.
+  If necessary, run:
++
+----
+zypper install -- -kernel-default-base +kernel-default
+----
+////
+
+
+=== Limitations
+
+{CoCo}  Attestation is currently implemented only as a technological preview. 
+//OM: (Do we state this explicitly in the documentation?)
+
+
+
+== Use {CoCo} in {productname}
+
+.Procedure: Enabling Attestation
```

Comment:
You need also another procedure: "Enable the attestation container"

As only a few people have the hardware and the need for CoCo, the attestation container is disabled by default.
@lucidd can describe how to enable the container.
This is a kind of "Pre Requirement" for this feature

Reply:
The feature can be enabled during installation of susemanager with with `mgradm install podman`. To enable it either  this bit has to be present in the mgradm.yaml confg
```yaml
coco:
    replicas: 1
```
or alternatively the corresponding command line parameter --coco-replicas 1 needs to be passed.
The current plan for enabling disabling the container after installation is something like this `mgradm scale --coco-replicas 1` 1 for enabled 0 for disabled. In the future that number should allow more then one container replica too.